lib/galaxy/webapps/galaxy/api/tools.py

```diff
@@ -205,7 +206,7 @@ def test_data(self, trans, id, **kwd):
         def json_encodeify(obj):
             if isinstance(obj, OrderedDict):
                 return dict(obj)
-            elif isinstance(obj, map):
+            elif inspect.isclass(map) and isinstance(obj, map):
                 return list(obj)
```

Comment:
@mvdbeek This also shouldn't be needed any more after https://github.com/galaxyproject/galaxy/pull/9416 ?

I think we may just drop the method.

Reply:
Yep, done that. Targeting dev for more extended test coverage, we can backport after merge